packages/jest/package.json

```diff
@@ -44,7 +44,7 @@
   "dependencies": {
     "@angular-devkit/architect": ">=0.1500.0 < 0.1600.0",
     "@angular-devkit/core": "^15.0.0",
-    "jest-preset-angular": "12.2.6",
+    "jest-preset-angular": "13.0.1",
```

Comment:
13.1.0 expanded support for ng16
https://github.com/thymikee/jest-preset-angular/blob/main/CHANGELOG.md#1310-2023-05-06

Is there a reason to pin the version of jest-preset-angular, instead of:
`"jest-preset-angular": "^13.1.0",`

Reply:
No. You could’ve noticed the PR was created before the 13.1 was released.